### PR TITLE
Allows for specifying test verbosity via Maven

### PR DESCRIPTION
**Pull request by ememisya**
ex:
mvn -Dverbosity=10 test
See: Level of Verbosity at https://maven.apache.org/surefire/maven-surefire-plugin/examples/testng.html

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -68,6 +68,7 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <checkstyle.dir>${basedir}/src/main/checkstyle</checkstyle.dir>
     <assembly.dir>${basedir}/src/main/assembly</assembly.dir>
+    <verbosity>0</verbosity>
```
Comment (dfish3r) [inline pom.xml]: Not sure this rises to the level of needing a property.

File: pom.xml [hunk 2]
```diff
@@ -175,6 +176,12 @@
         <configuration>
           <parallel>methods</parallel>
           <threadCount>10</threadCount>
+          <properties>
+            <property>
+              <name>surefire.testng.verbose</name>
+              <value>${verbosity}</value>
```
Comment (dfish3r) [inline pom.xml]: Just set it to zero here, no property.

Comment (ememisya) [inline pom.xml]: This is for compatibility with the default value, and sake of simplicity.  One can just clone and set mvn -Dverbosity=10 test, wouldn't you agree?

**Comment by serac** [comment]
Can you explain the rationale for this property? Do you want more or less test output?

**Comment by ememisya** [comment]
The default characteristic of the test goal seems to be no output except the results of the tests after they have been performed.  I wanted the ability to see the tests as they are being executed and with which certificates and so on, which lead me to testng's verbosity feature.  By implementing the property we can still keep the current behaviour until the developer specifies the verbosity parameter between 1-10 depending on the level of information they wish to see during testing.

**Comment by serac** [comment]
Understood, thanks. I think this is a beneficial change, though I might want to see it driven by a profile instead of a property. It should default to a reasonable value (0?) such that it's not required to be set on the command line. A profile is one straightforward way to accomplish that.

**Comment by dfish3r** [comment]
It doesn't bother me if it remains a property, but it should have a more descriptive name.
Like ${testng.verbosity}

**Comment by serac** [comment]
My only requirement is that it's not necessary to specify -Dproperty.name=X on the command line to build.

**Pull request merged**